receiver/cloudflarereceiver/testdata/cert/CAroot.crt

```diff
@@ -0,0 +1,17 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
If at all possible, please add this module to the Makefile's [cert list ](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/cf47a1bfbfadcfd40a94e59a9da157187a3a420d/Makefile#L353)and generate these files using that target.

Reply:
Do we still need this file, now that we have `ca.crt`?

Reply:
Nope, looks like I didn't clean up everything, will do.